### ISSUE TITLE
Connector for Snowflake

### ISSUE THREAD

**Issue by superbeagle** [camunda/connectors#1116]
__Is your feature request related to a problem? Please describe.__

Many financial institutions have been asking for integration with Snowflake:

https://www.snowflake.com/en/

__Describe the solution you'd like__

Inbound and Outbound connectors for Snowflake. From my understanding it works in a similar way to a database interaction

__Describe alternatives you've considered__

Maybe I'll give it a go and try building one myself :) 

__Additional context__

**Comment by bastiankoerber** [comment]
@superbeagle do you have any use case in mind? 
I had document in the past  to support the following API;: 

Supported Operations:
Create stage
Delete warehouse
List table columns DDL
Update table
Update warehouse
Create database
Create schema
Create table
Create warehouse
Custom query
Delete database
Delete files from stage
API Reference:
https://docs.snowflake.com/en/developer-guide/develop-apps-extensions

‌

**Comment by superbeagle** [comment]
Right, from my understanding it works very much like a database. A number of folks have been asking for it but nothing more specific than 'We'd like to be able to connect to Snowflake'. The above list should do the trick
